pkgs/development/tools/analysis/makefile2graph/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchFromGitHub, makeWrapper, bash, gnumake }:
+
+stdenv.mkDerivation rec {
+  name = "makefile2graph-2018-01-03";
+
+  src = fetchFromGitHub {
+    owner = "lindenb";
+    repo = "makefile2graph";
+    rev = "61fb95a5ba91c20236f5e4deb11127c34b47091f";
+    sha256 = "07hq40bl48i8ka35fcciqcafpd8k9rby1wf4vl2p53v0665xaghr";
+  };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  buildPhase = ''
+    make
+  '';
+
+  installPhase = ''
+    make install prefix="$out"
+  '';
```

Comment:
Could you delete `buildPhase` and `installPhase`, and add `makeFlags = [ "prefix=$(out)" ];`?

Reply:
This is done!